test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -118,8 +112,23 @@ def test_listen_create_request(self):
                 "durable-subscription-name": "events",
             },
         )
+        assert stomp_connection.get_listener("bg_listener") == listener
```

Comment:
Reordered so that variable creation and use are close to each other. Also added some additional asserts so if this fails because of a step before the last, it will be clear where the failure happened.